src/main/java/model/data/value/DataValue.java

```diff
@@ -15,4 +16,12 @@
 
 	@Override
 	public abstract int hashCode();
+	
+	public boolean isNull() {
+		return isNull;
+	}
+	
+	public void setNull(boolean nul) {
```

Comment:
Zou je deze niet protected maken? Het lijkt me sterk dat het slim is om alles hiermee te laten zooien.